### DISCUSSION TITLE
Touchpad scrolling speed

### DISCUSSION THREAD

**Discussion by Unwrapped3** [hyprwm/Hyprland#3982] [Q&A] [answered]
Hi everyone! The default scrolling speed on my touchpad was way too high so I've added scroll_factor = 0.2 to the hyprland.conf, as suggested [here](https://www.reddit.com/r/hyprland/comments/121s3es/the_scroll_sensitivity_of_my_touchpad_is_way_too/).

It made scrolling normal everywhere but it is way too slow in the terminal now (zsh/kitty). 

Is there a way to fix this? 

Thanks in advance!

**Comment by Readf0x** [comment]
Bumping this because I have the exact same problem

**Comment by fufexan** [comment]
Looks like Kitty doesn't have a built-in way of changing the scroll amount, so your options would be
1. Use a different terminal. Wezterm handles this well.
2. Try `input:scroll_points` for adaptive scrolling. https://wiki.hyprland.org/Configuring/Variables/#input

**Comment by Oskar-Idland** [comment, marked as answer]
I had the same problem. Adding `touch_scroll_multiplier 10.0` in my `~/.config/kitty/kitty.conf` was enough. Just change the number as needed

**Reply by Readf0x** [reply]
thank you!

**Reply by hrasekj** [reply]
@Unwrapped3 Should be maked answered. This worked for me too.

**Comment by bitnerry** [comment]
I was also having the same problem with my alacritty terminal. I just make some changes in my `~/.config/alacritty/alacritty.toml` putting `multiplier = 10` under the **[scrolling]** section. 

were `multiplier = 10` mean that it will scroll 10lines per scrolling event
